<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRewardWidget</class>
 <widget class="QWidget" name="CRewardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>35</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0" colspan="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="cb_rewardType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cb_object">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="sp_rewardCount">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="cb_psyState">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>4</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
